client/src/components/Footer.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "../styles/Footer.css";
+import CYFLogo from "../assets/CYF.jpeg";
+import { Divider, Grid, Image, Segment } from "semantic-ui-react";
+
+const Footer = () => {
+  return (
+    <div className="footer">
+      <Grid divided columns={3}>
+        <Grid.Column>
+          {" "}
+          <div className="team-container">
+            <p>Min-Imece-Team</p>
+            <span>Samuel</span>
+            <span>Hacer</span>
+            <span>Fatma</span>
+            <span>Mawaddah</span>
+            <span>Natalia</span>
+          </div>
+        </Grid.Column>
+        <Grid.Column>
+          <h3>
```

Comment:
```suggestion
          <p>
```

This looks like a sentence rather than a title. A sentence should use the paragraph element `p` — let's update this